GOVERNANCE.md

```diff
@@ -68,6 +68,8 @@ If a vote is taken during a meeting, the follow rules will be followed:
 * A participating company, or nonaffiliated individual, attains voting rights
   by having any of their assigned representative(s) attend 3 out of the last
   4 meetings. They obtain voting rights after the 3rd meeting, not during.
+  If the primary representative can not attend a meeting, his/her
+  delegate's attendance is counted.
```

Comment:
Cathy - I agree with the idea but its kind of awkward to have this sentence separate from the one on line 68.
Would you be ok with this instead:
A participating company, or non-affiliated individual, attains voting rights
by having their assigned representative(s), in aggregate, attend 3 out of
the last 4 meetings.